src/core/api/utils.go

```diff
@@ -64,7 +64,7 @@ func SyncRegistry(pm promgr.ProjectManager) error {
 	}
 
 	if len(reposToAdd) > 0 {
-		log.Debugf("Start adding repositories into DB... ")
+		log.Infof("Start adding repositories into DB %v ... ", len(reposToAdd))
```

Comment:
How is this change necessary for switching CI?

Reply:
This message should be info level, or it will be hard to known how many repositories being imported, so I change it  by the way.
Of course this is not for CI